### DISCUSSION TITLE
Assign existing Todo to a different Project

### DISCUSSION THREAD

**Discussion by compumatter** [Leantime/leantime#974] [Q&A]
If I have created a project in one client (along with its data) that I feel at some point is better suited to another is there a way to move a project to a different client.  For instance in the example below moving the Overheating NVME project to CompuMatter.

![image](https://user-images.githubusercontent.com/121611/191105070-3f988913-7aaf-4747-8907-6ab7e592c9ec.png)

On a related note, is there a way to move Todo's from one project to another project?

Thanks.

**Comment by ftrueck** [comment]
Click on the project name.
In the left side menu click the last link: project settings.
In the customer dropdown select a new customer.

**Reply by ftrueck** [reply]
Moving todos to other projects in the leantime app is currently not possible.
However: If you know how to manage a mysql DB you can assign a new project id to the todos.

**Reply by compumatter** [reply]
Thank you.  Our purpose is to provide this product to our clients as a way to manage their projects within our server.  So we are really needing a layperson useable feature.  We are testing Leantime within our own work environment along with Wekan, Openproject and a few others.  We really like Leantime but this is a feature missing that comes up often as needed.

**Reply by gloriafolaron** [reply]
@compumatter Would templates also work in this situation? ex: To Do templates or Project templates?
